include/seqan3/argument_parser/auxiliary.hpp

```diff
@@ -108,6 +108,8 @@ struct argument_parser_meta_data // holds all meta information
      *        "./my_read_mapper -s 3 --my_flag path/infile1").
      */
     std::vector<std::string> examples;
+    //!\brief Whether to always deactivate the version check (attention: Your users will never receive update infos).
+    bool deactivate_version_check{false};
```

Comment:
I thought you renamed this above?

Reply:
This should not appear here at all since it is a constructor argument. thanks